edi_oca/models/edi_exchange_consumer_mixin.py

```diff
@@ -291,3 +291,45 @@ def _edi_set_origin(self, exc_record):
     def _edi_get_origin(self):
         self.ensure_one()
         return self.origin_exchange_record_id
+
+    def _edi_send_via_edi(self, exchange_type):
+        exchange_record = self._edi_create_exchange_record(exchange_type)
+        exchange_record.action_exchange_generate_send()
+        msg = _("EDI auto: output generated.")
```

Comment:
This is not only generated but generated and sent.
I wonder if we need this custom message...
The backend will call `notify_action_complete` when is done...